src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -112,7 +112,8 @@ const formConfig = {
         // paymentInformation:
         specialCircumstances: {
           title: 'Special Circumstances',
-          path: 'special-circumstances',
+          path: 'review-veteran-details/special-circumstances',
+          depends: formData => formData.prefilled,
```

Comment:
Not sure I understand the implication of this - does this mean that we don't want to show the page at all if the user has any prefilled data? I'm probably just missing something since this page seems to work as expected when paging through the form.

Reply:
There are two versions of the first chapter, verified and unverified. These depends ensure that users are routed to the appropriate version.